### PR TITLE
Clean up ng comp lib docs

### PR DESCRIPTION
**Pull request by huayunh**
Changes proposed in this Pull Request:
- Change all the `scorecard` to `score-card`
- Pretty everything
- Clean up the docs for HTML escape chars
- Double-check component API readme to have them line up with what's implemented in the code and that of the React API 
- Bump up version to 2.0.0

Waiting for an update in ng theme

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,31 @@
 # Change Log
 
+## v2.0.0
+
+-   `ChannelValue`'s value attribute now accepts both `string` type and `number` type.
+    _Breaking Changes_
+-   `pxb-scorecard` is now `pxb-score-card`.
```
Comment (emclaug2) [inline CHANGELOG.md:7]: Scorecard and info list item didn't exist in 1.3.0, can you update your breaking changes section?

Comment (emclaug2) [reply CHANGELOG.md:7]: Spacer is new too, could you add it as well?

File: components/src/core/empty-state/empty-state.component.scss [hunk 2]
```diff
@@ -30,3 +30,7 @@
         }
     }
 }
+
+h2 {
```
Comment (emclaug2) [inline components/src/core/empty-state/empty-state.component.scss:34]: You shouldn't need this rule anymore, there's class for pxb-empty-state-title that has margin-bottom: 0 on it.

Comment (emclaug2) [reply components/src/core/empty-state/empty-state.component.scss:34]: Avoid styling directly on elements.   Our components aren't using style encapsulation anymore so these styles will bleed into other components.  If we stick to using classes this shouldn't be an issue.

File: demos/showcase/src/app/app.component.scss [hunk 3]
```diff
@@ -20,26 +20,29 @@
...
+        padding: 0px !important;
     }
     .mat-list-item {
         height: 48px !important;
     }
     .mat-list-text {
         padding-left: 0px !important;
     }
+    .pxb-score-card .pxb-score-card-header {
+        background-color: map-get($pxb-blue, 500);
```
Comment (emclaug2) [inline demos/showcase/src/app/app.component.scss:41]: This'll be set by the pxblue blue theme, optional removal.

File: docs/ChannelValue.md [hunk 4]
```diff
@@ -1,42 +1,54 @@
...
+| Attributes | Description                                    | Type              | Required | Default   |
+| ---------- | ---------------------------------------------- | ----------------- | -------- | --------- |
+| fontSize   | The size of the font                           | `string`          | no       | 'inherit' |
+| prefix     | Show units before the value                    | `boolean`         | no       | false     |
+| units      | The text to display for the units (light text) | `string`          | no       |           |
+| value      | The the value (bold text) to display           | `string | number` | yes      |           |
 
 </div>
+
+Child element:
```
Comment (joebochill) [inline docs/ChannelValue.md:50]: We should have consistent language across readmes for this...some say "Child element" others say "Child element with attributes"...Is there a distinction that we should explain? Does this particular child need to have an attribute to be picked up?

File: docs/ScoreCard.md [hunk 5]
```diff
@@ -9,8 +9,8 @@ Card component that calls attention to particular values.
 
 ## Usage
 
-<div style="align-items: center; text-align: center; width: 100%; display: flex; justify-content: space-evenly; margin-bottom: 20px">
-    <img width="80%" alt="Score Card with labels on different parts" src="./images/scoreCardAnatomy.png"><br/>
+<div style="width: 100%; text-align: center">
+    <img width="100%" style="max-width: 800px" alt="Empty State Anatomy" src="./images/scoreCardAnatomy.png">
```
Comment (joebochill) [inline docs/ScoreCard.md:13]: need to fix the alt text here

File: demos/storybook/stories/hero/different-image-types.component.ts [hunk 6]
```diff
@@ -1,57 +1,57 @@
...
 const iconSet = require('@pxblue/icons-svg/icons.svg');
 const Trex = require('../../assets/trex.png');
 
 @Component({
     selector: 'image-types-story',
     template: `
         <pxb-hero-banner>
-            <pxb-hero label='SVG' value='normal' iconSize='normal' [iconBackgroundColor]='colors.white[50]'>
-                <mat-icon svgIcon='px-icons:grade_a' primary [style.color]='colors.blue[500]'></mat-icon>
+            <pxb-hero label="SVG" value="normal" iconSize="normal" [iconBackgroundColor]="colors.white[50]">
```
[the remark this answers is not in the document]
Comment (huayunh) [reply demos/storybook/stories/hero/different-image-types.component.ts:14]: I guess it is considering that as an HTML code, and the HTML convention is double quote

File: docs/ChannelValue.md (hunk 4 above)
Comment (huayunh) [reply docs/ChannelValue.md:50]: Child element:
Usually only one child element is needed, no attributes needed

Child element with attributes:
Multiple child elements are picked up by the component, and they all have attributes

Tell me how you would like to word these.

File: demos/showcase/src/app/app.component.html [hunk 7]
```diff
@@ -128,7 +128,7 @@
     <div class="error">
         <pxb-info-list-item title="Output Voltage" divider="full" dense="true">
             <mat-icon icon svgIcon="px-icons:voltage_circled"></mat-icon>
-            <div right-component>
+            <div right-component [style.color]="colors.black[500]">
```
[the remark this answers is not in the document]
Comment (huayunh) [reply demos/showcase/src/app/app.component.html:131]: the class="error" will enforce the red 500 color on the right component, resulting in a weird red comma thing

File: docs/ScoreCard.md (hunk 5 above)
Comment (huayunh) [reply docs/ScoreCard.md:13]: Good catch

**Comment by huayunh** [comment]
@joebochill I am afraid that I have to add the scorecard list padding back; have to do that to match with React style

Let me know what you think and then I will change those examples in the storybook.

With padding: 
image

Without padding: 
image

File: docs/ChannelValue.md (hunk 4 above)
Comment (joebochill) [reply docs/ChannelValue.md:50]: Okay that makes sense. Maybe we just leave the attribute column blank for the case when there's no specific selector/attribute used to transclude the child.

**Comment by joebochill** [comment]
@huayunh the padding looks fine. The egregious one was in the action row at the bottom because it affected the vertical centering of the text and icon.

File: demos/storybook/stories/hero/different-image-types.component.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (joebochill) [reply demos/storybook/stories/hero/different-image-types.component.ts:14]: Yeah, it looks like the other files do this too. Looks fine :)

File: docs/ChannelValue.md (hunk 4 above)
Comment (huayunh) [reply docs/ChannelValue.md:50]: I am using "(child)" right now

**Comment by huayunh** [comment]
> @huayunh the padding looks fine. The egregious one was in the action row at the bottom because it affected the vertical centering of the text and icon.

I changed the action row to info list item, so that just went away.

**Pull request merged**